### ISSUE TITLE
`cbcm-browser` and `chrome-os_device` addresses are  not properly handled in `gam update cigroups <GroupEntity> delete|sync|update`

### ISSUE THREAD

**Issue by taers232c** [GAM-team/GAM#1722]
The id:prefix is not properly stripped from the address

**Comment by jay0lee** [comment]
should be fixed now?

**Comment by jay0lee** [comment]
This still isn't working properly. We currently have:

```
def convertUIDtoEmailAddress(emailAddressOrUID, cd=None, emailTypes=None,
                             checkForCustomerId=False, ciGroupsAPI=False, aliasAllowed=True):
  if ciGroupsAPI:
    normalizedEmailAddressOrUID = normalizeEmailAddressOrUID(emailAddressOrUID, ciGroupsAPI=ciGroupsAPI)
    if normalizedEmailAddressOrUID.startswith('cbcm-browser.') or normalizedEmailAddressOrUID.startswith('chrome-os-device.'):
      return normalizedEmailAddressOrUID
  email, _ = convertUIDtoEmailAddressWithType(emailAddressOrUID, cd, emailTypes,
                                              checkForCustomerId, ciGroupsAPI, aliasAllowed)
  return email
```
But we're not short circuiting these special strings in time, The call to `normalizeEmailAddressOrUID()` before the check is appending domain name to the strings.

**Comment by taers232c** [comment]
Thanks, will fix.

Ross
----
Ross Scroggs
***@***.***

**Comment by taers232c** [comment]
The code is expecting a id:prefix just list for user IDs.
<CBCMBrowser> ::= id:cbcm-browser.<DeviceId>
<ChromeOSDevice> ::= id:chrome-os-device.<DeviceId>
I'll update the code to allow for no id:

Ross
----
Ross Scroggs
***@***.***

**Comment by taers232c** [comment]
Fixed in 7.02.01
